src/app/weather.model.ts

```diff
@@ -32,4 +32,28 @@ export interface ICoordinates {
 
 export interface IPosition {
   coords: ICoordinates;
+}
+
+export interface IWidget {
+  cityName: string;
+  temperature: number;
+  weatherCondition: string;
+  maxTemperature: number;
+  minTemperature: number;
+  description: string;
+}
+
+export interface IOpenWeatherMapResponse {
+  name: string;
+  main: {
+    temp: number;
+    temp_max: number;
+    temp_min: number;
+  };
```

Comment:
Need a separate interface for the field `main`. 

P.S: Easy rule if you have something more difficult than simple type - create an interface for it.